memegen/stores/image.py

```diff
@@ -22,10 +17,10 @@ def __repr__(self):
         return "<Meme(id='%s', key='%s')>" % (self.id, self.key)
 
     def save(self):
-        SA.add(self)
+        db.session.add(self)
```

Comment:
I usually call `session.commit()` in `Model.save()` since the word "save" implies that stuff is actually getting saved, not just just added to a queue of stuff to maybe save.